cmd/stack_yaml_lint.go

```diff
@@ -0,0 +1,157 @@
+// Copyright © 2019 IBM Corporation and others.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"bufio"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+type StackDetails struct {
+	Name        string            `yaml:"name"`
+	Version     string            `yaml:"version"`
+	Description string            `yaml:"description"`
+	License     string            `yaml:"license"`
+	Language    string            `yaml:"language"`
+	Maintainers []StackMaintainer `yaml:"maintainers"`
+}
+
+type StackMaintainer struct {
+	Email string `yaml:"email"`
+}
+
+var DefaultFound bool
```

Comment:
Hate to be a pest: can this be renamed `defaultStackFound`? Also - does it need to be capitalized (i.e. visibile from outside the cmd package)??

Reply:
Changed to `defaultTemplateFound` as I thought that might be more accurate